driver/bpf/fillers.h

```diff
@@ -2381,44 +2381,45 @@ FILLER(proc_startupdate, true)
 		args_len = 0;
 	}
 
-	if (args_len > 0) {
-		int exe_len;
+	int exe_len;
```

Comment:
Can you leave a a self review of the changes?

Reply:
It turns out that `args_len` could be zero (I didn't look at the root cause though) but at the same time the `exe` could be valid. So here I always extract `exe_len` to check whether or not we have to actually read `exe`. Without this patch, with the new tests we are missing the `exe` param that is correctly extracted in the other drivers.